src/plugins/importFromJSONAction/ImportFromJSONAction.js

```diff
@@ -170,9 +276,16 @@ export default class ImportAsJSONAction {
    * @param {Object} objTree
    */
   async _importObjectTree(domainObject, objTree) {
```

Comment:
can we switch these to actual private methods without ruining the unit tests?

Reply:
Changing these to private methods cause issues with the [way we're returning](https://github.com/nasa/openmct/blob/41d0b5acce3f99c0f8086d2b19bd1dbe4360b640/src/plugins/importFromJSONAction/ImportFromJSONAction.js#L376) from `_showForm`. E.g., changing to `#importObjectTree` causes:
```
ImportFromJSONAction.js:77 Uncaught (in promise) TypeError: Receiver must be an instance of class ImportAsJSONAction
    at ImportAsJSONAction.onSave (ImportFromJSONAction.js:77:1)
    at eval (ImportFromJSONAction.js:376:1)
```